dirAlias.js

```diff
@@ -1,7 +1,9 @@
 const path = require('path');
 const fs = require('fs');
 
-const appDirectory = fs.realpathSync(process.cwd());
+let appDirectory = fs.realpathSync(process.cwd());
+appDirectory = appDirectory.replace('/ws-renderer', '');
```

Comment:
quick hack to get directory aliases to work from the context of the nextjs subdirectory